src/renderer/Extensions/Base64Conversion/Converter.tsx

```diff
@@ -0,0 +1,76 @@
+import { useContextBridge } from "@Core/Hooks";
+import { Textarea } from "@fluentui/react-components";
+import { useEffect, useState } from "react";
+import { InvocationArgument } from "../../../main/Extensions/Base64Conversion/InvocationArgument";
```

Comment:
Don't import files across the "main" and "renderer" folders.

The rules are:
- A file in the "renderer" directory is only allowed to import files from "renderer" and "common".
- A file in the "main" directory is only allowed to import files from "main" and "common".

I suggest moving the shared code to the common folder then you can import them like this:

```typescript
import { MyStuff } from "@Common/Extensions/MyExtension/MyStuff";
```

Sorry, you couldn't know these things as they're not documented (yet).